samples/net/sockets/echo_client/Kconfig

```diff
@@ -46,4 +46,12 @@ config NET_SAMPLE_IFACE3_VLAN_TAG
 	  Set VLAN (virtual LAN) tag (id) that is used in the sample
 	  application.
 
+config NET_SAMPLE_PSK_HEADER_FILE
+	string "Header file containing PSK"
+	default "dummy_psk.h"
+	depends on MBEDTLS_KEY_EXCHANGE_PSK_ENABLED
+	help
+	  Enter name of a header file containing a
```

Comment:
I'd say word "Enter" is superfluous here, we don't use such intro-verbs in help texts.

Reply:
Removed.